src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/HotReload/IProjectHotReloadSession.cs

```diff
@@ -20,8 +21,22 @@ public interface IProjectHotReloadSession
         /// <summary>
         /// Starts the Hot Reload Session.
         /// </summary>
+        /// <remarks>
+        /// TODO: remove when Web Tools is no longer calling this method.
+        /// </remarks>
+        [Obsolete("This should no longer be used; please use StartSessionAsync(bool, CancellationToken) instead.", false)]
         Task StartSessionAsync(CancellationToken cancellationToken);
```

Comment:
>StartSessionAsync [](http://example.com/codeflow?start=13&length=17)

can this be removed since we're still in preview, or how will web tools know to not use this?

Reply:
I'm in communication with the Web Tools team. We can't remove this method without breaking their Hot Reload scenarios, so we're leaving it in until they have a chance to move off of it. Once we do I'll remove it entirely.